### ISSUE TITLE
Is there a way to get my index.ejs to compile to index.ejs?

### ISSUE THREAD

**Issue by kajweb** [trapcodeio/vite-plugin-ejs#14]
I am using koa to output index, and then I want to get a similar ejs file to be output by koa. Can you give me some advice?

![image](https://github.com/trapcodeio/vite-plugin-ejs/assets/2993320/02a444aa-d4ef-4cdd-bdfa-ed86d03b11d3)

Then, output this page through koa-views (server)
![image](https://github.com/trapcodeio/vite-plugin-ejs/assets/2993320/238c7a80-c65a-4d55-8c59-7eeb303e59df)

The interface you ultimately want to render the code on the client
![image](https://github.com/trapcodeio/vite-plugin-ejs/assets/2993320/e5bf38d6-1dac-457c-b461-4c1097a7bc45)

**Comment by ticlaudeberube** [comment]
I have a similar need using Express. Actullay I am looking to use Vite instead of NodeMon. The site is a standard back-end app used for usability purposes, so an SPA is not possible. Content would need to be compiled.

**Comment by joseph-hanvy-sophos** [comment]
Yeah, I have a similar requirement, I want to output an `.ejs` that can be templated

**Comment by reneroth** [comment]
Guys, it's a 60 line vite plugin. Just look at it and make your own :)

**Comment by trapcodeio** [comment]
My apologies for the very late response; I thought I had already replied.
The feature you are requesting is out of the scope of this plugin. @kajweb
